R/cluster.R

```diff
@@ -387,31 +372,113 @@ getCluster <- function(clusterName) {
 
   nodes <- rAzureBatch::listPoolNodes(clusterName)
 
-  currentNodeCount <- 0
   if (!is.null(nodes$value) && length(nodes$value) > 0) {
-    nodesStatus <- .processNodeCount(nodes)
-
-    currentNodeCount <- nodesStatus$currentNodeCount
-    nodesWithFailures <- nodesStatus$nodesWithFailures
+    nodesInfo <- .processNodeCount(nodes)
+    nodesState <- nodesInfo$nodesState
+    nodesWithFailures <- nodesInfo$nodesWithFailures
+
+    if (verbose == TRUE) {
+      cat("\nnodes:", fill = TRUE)
+      cat(sprintf("\tidle:                %s", nodesState$idle), fill = TRUE)
+      cat(sprintf("\tcreating:            %s", nodesState$creating), fill = TRUE)
+      cat(sprintf("\tstarting:            %s", nodesState$starting), fill = TRUE)
+      cat(sprintf("\twaitingforstarttask: %s", nodesState$waitingforstarttask), fill = TRUE)
+      cat(sprintf("\tstarttaskfailed:     %s", nodesState$starttaskfailed), fill = TRUE)
+      cat(sprintf("\tpreempted:           %s", nodesState$preempted), fill = TRUE)
+      cat(sprintf("\trunning:             %s", nodesState$running), fill = TRUE)
```

Comment:
probably worth adding and 'other' state for all other states, since this could cause confusion.

Reply:
added 'other' state